### ISSUE TITLE
Codemagic Deployment to App Store not working

### ISSUE THREAD

**Issue by JanikHalder** [FlutterFlow/flutterflow-issues#3052]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Codemagic Deployment to App Store not working

### Expected Behavior

Deploy to the App Store

### Steps to Reproduce

1. Deploy to the App Store

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4gjvLq34V2obhf7qrAcu9VuDoRNkQjUL00sNobbxAjIoDoPJkUac78YltDXfG8YXx5KEWKmnMFpPTVj+DlL/IBG0yZR7o61YdyFgndXk+/aoSlCLuOXGt6QO1VGX6X0raskSNvOtR1LUYG7VyMGe7KThXsHqH6Zwh9f6PHaOI=

### Context

Deploy to the App Store

### Visual documentation

<img width="1296" alt="Bildschirmfoto 2024-06-03 um 16 53 41" src="https://github.com/FlutterFlow/flutterflow-issues/assets/95503532/13637c5e-885f-4021-a4f2-fe7d2294a602">

### Environment

```markdown
- FlutterFlow version: 4.1.54
- Platform: MacOS and on Web
- Browser name and version: Chrome
- Operating system and version affected:
```

### General

- [ ] I was collaborating with other users in the same project prior to data loss
- [ ] I navigated between different project views leading up to data loss
- [ ] The UI was running very slowly prior to data loss
- [ ] A crash or other bug occurred prior to data loss
- [ ] Data was not saved in a snapshot
- [ ] I was using flutterflow from multiple devices
- [ ] I was using flutterflow from multiple platforms (e.g. both web and MacOS Desktop app)

### Relative to the time the changes were made, data was lost within

- [ ] 1 minute
- [ ] 1 hour
- [ ] 1 day
- [ ] 1 week

### When following my steps to reproduce, data loss happens

- [ ] ~100% of the time
- [ ] ~50% of the time
- [ ] ~10% of the time
- [ ] ~1% of the time
- [ ] Very rarely or only encountered once

**Comment by jlsimmo** [comment]
I have the same issue since about 1pm UTC

**Comment by Wojtek-G** [comment]
> Same here

It's working now for me.

**Comment by KeepTrackMed** [comment]
Not able to build to either store with the same message June 4th for us here too.

**Comment by msusviela** [comment]
Hello @jlsimmo, @KeepTrackMed and @JanikHalder. Lately Codemagic has been experiencing some degraded services. This could be related to this error. Is the issue still persistent for you?

**Comment by jlsimmo** [comment]
Yes, that's probably it. I managed to do one build today but then started getting the same issue on both my active projects

**Comment by lleo132** [comment]
Same problem here

**Comment by itsstevenliu** [comment]
Same problem here :(

**Comment by emanuel-braz** [comment]
same here

**Comment by shefa** [comment]
+1
Can we get the desktop app to do this build +push locally instead?
This is so annoying!

**Comment by sanjayelate** [comment]
Same issue here with same error message. Happening on both desktop and web. 

Can someone at @FlutterFlowEng at least let us know if it's an issue at their end or if they're waiting for code magic?

**Comment by jlsimmo** [comment]
It's codemagic...

https://status.codemagic.io/
